predict.py

```diff
@@ -0,0 +1,105 @@
+"""
```

Comment:
Could you add a license header to this file, similar to https://github.com/facebookresearch/SWAG/blob/main/imagenet_1k_eval.py?

Reply:
Just added!